app/components/authorization/authorization.service.js

```diff
@@ -2,46 +2,46 @@
 
 angular
   .module('mentors4me')
-  .factory('authorizationService', function($cookies, USER_ROLES) {
+  .factory('authorizationService', function($cookies, USER_ROLES, Constants) {
       var authorizationService = {};
 
       authorizationService.isAuthorized = function(userRoles) {
-        if (this.isLoggedIn() && userRoles.indexOf($cookies.get("userRole")) !== -1) {
+        if (this.isLoggedIn() && userRoles.indexOf($cookies.get(Constants.USER_ROLE)) !== -1) {
           return true;
         }
         return false;
       }
 
       function isNotEmpty(element) {
-        if (element !== "") {
+        if (element !== Constants.EMPTY) {
```

Comment:
I Don't think we need constant here. Also you can put it like: `return element !== ''`